charts/k8s-watcher/README.md

```diff
@@ -152,7 +152,9 @@ The following table lists the configurable parameters of the chart and their def
 | `watcher.resources.csiDriver`                      | Enables watching CSIDriver                                                                                                                                                    | `true`                              |
 | `watcher.resources.csiNode`                        | Enables watching CSINode                                                                                                                                                      | `true`                              |
 | `watcher.resources.csiStorageCapacity `            | Enables watching CSIStorageCapacity                                                                                                                                           | `true`                              |
-| `watcher.resources.volumeAttachment`               | Enables watching VolumeAttachment                                                                                                                                             | `true`                              |
+| `watcher.resources.volumeAttachment`               | Enables watching VolumeAttachment                                                                                                                       | `true`                              |
+| `watcher.resources.allowReadAll`               | Enables reading all the resources in the cluster                                                                                      | `true`                              |
+| `watcher.resources.customReadAPIGroups`               | A whitelist of allowed resources to read from cluster - each array element should a string which represents the group name                                                                                    | `[]`                              |
```

Comment:
1. this sentence is grammarly incorrect - `each array element should a string`
2. I would rephrase this to `A list of custom API groups to allow read access to`
3. also I'd keep indentation of the `|` mark on this line and on line 156